app/adopt_a_paw/page.tsx

```diff
@@ -157,191 +164,189 @@ const AdoptAPaw = () => {
-            <button
-              key={category}
-              onClick={() => handleCategoryChange(category)}
-              type="button"
-              className="text-white bg-blue-700 hover:bg-blue-800 focus:outline-none focus:ring-2 focus:ring-blue-300 font-medium rounded-full text-sm px-5 py-2.5 text-center me-2 mb-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
-            >
-              {category}
-            </button>
-          ))}
-        </div>
-
-        {/* main-area */}
-        <div className="flex flex-wrap gap-4 pt-6 justify-center">
-          {isLoading && (
-            <div className="loading-container">
-              <div className="loading-animation" />
-              <p>Loading pet data...</p>
-            </div>
-          )}
-
-          {pets &&
-            filterPetsByCategory()
-              .slice(0, visiblePets)
-              .map((pet) => (
-                <div
-                  key={pet.petId}
-                  className="relative flex flex-col m-2 text-gray-700 bg-white shadow-md rounded-xl sm:w-1/2 md:w-1/3 lg:w-1/3 xl:w-1/5 max-w-xs sm:max-w-sm md:max-w-md lg:max-w-lg xl:max-w-xl"
-                  data-testid="item-container"
-                >
-                  <div className="w-full p-3">
-                    <div className="relative h-44 overflow-hidden text-white rounded-xl bg-blue-gray-500 shadow-blue-gray-500/40">
-                      <img
-                        className="rounded-lg"
-                        src={pet.image}
-                        alt={pet.name}
-                        data-testid="item-image"
-                      />
-                    </div>
+    <div className="relative flex flex-col items-center justify-center bg-slate-50">
+      {/* filter */}
+      <div className="p-20" />
+      <div className="flex flex-wrap justify-center sm:justify-start space-x-4">
+        {categories.map((category) => (
+          <button
+            key={category}
+            onClick={() => dispatch(setCategoryState(category))}
+            type="button"
+            className="text-white bg-blue-700 hover:bg-blue-800 focus:outline-none focus:ring-2 focus:ring-blue-300 font-medium rounded-full text-sm px-5 py-2.5 text-center me-2 mb-2 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800"
+          >
+            {category}
+          </button>
+        ))}
+      </div>
 
-                    <div
-                      className="p-3 "
-                      onClick={() => toggleCardDetailPopup(pet)}
+      {/* main-area */}
+      <div className="flex flex-wrap gap-4 pt-6 justify-center">
+        {isLoading && (
+          <div className="loading-container">
+            <div className="loading-animation" />
+            <p>Loading pet data...</p>
+          </div>
+        )}
+
+        {pets &&
+          filterPetsByCategory()
+            .slice(0, visiblePets)
+            .map((pet) => (
+              <div
```

Comment:
This pet item content is too big here. Break it down as another component and import here.

Reply:
Done. I've created a new component called "PetItem.tsx"